x_xy/test_render.py

```diff
@@ -43,5 +45,33 @@ def test_animate(pytest=True):
             os.system(f"rm animation.{fmt}")
 
 
+def test_shapes():
+    sys_str = """
+<x_xy model="shape_test">
+    <options gravity="0 0 9.81" dt="0.01" />
+    <worldbody>
+        <geom type="sphere" mass="1" pos="0 0 0" dim="0.3" color="white" />
+        <geom type="box" mass="1" pos="-1 0 0" quat="1 0 1 0" dim="1 0.3 0.2" color="0.8 0.3 1 0" />
+        <geom type="cylinder" mass="1" pos="1 0 0.5" quat="0.75 0 0 0.25" dim="0.3 1" color="0.2 0.8 0.5" />
+        <geom type="capsule" mass="1" pos="0 0 -1" dim="0.3 2" />
+
+        <body name="dummy" pos="0 0 0" quat="1 0 0 0" joint="ry" />
+    </worldbody>
+</x_xy>
+    """
+
+    sys = load_sys_from_str(sys_str)
+
+    state = State.create(sys, q=None)
+
+    step_fn = jax.jit(dynamics.step)
```

Comment:
I would not jit here, it's okay but also doesn't do anything since we only execute the function once

Reply:
Interestingly enough the jax.jit actually provides a 10% speedup on my laptop (4.5s vs 5s), although on the CPU. I left it in for now.